test/utils/validators/forms/FieldValidatorSpec.scala

```diff
@@ -799,4 +799,34 @@ class FieldValidatorSpec extends WordSpec with MustMatchers {
     }
   }
 
+  "FieldValidator isValidName" should {
+
+    "return false" when {
+
+      "name is incorrect" in {
+
+        isValidName("Robert'); DROP TABLE Students;--") must be(false)
+        isValidName("James D@t") must be(false)
+      }
+    }
+
+    "return true" when {
+
+      "name is correct" in {
+
+        isValidName("Mathias d'Arras") must be(true)
```

Comment:
change the names to not to read as names please

Reply:
What do you mean?

I used acceptance criteria from ticket

Reply:
Done